### ISSUE TITLE
Problem with caching components

### ISSUE THREAD

**Issue by academici** [universal-vue/uvue#32]
**Describe the bug**
Сomponent is not cached. Getting an error: 
[vue-server-renderer] Component HelloWorld implemented serverCacheKey, but no cache was provided to the renderer.

**To Reproduce**
1. Create simple project
2. Create component example with 10s cache time
```vue
<template>
  <div class="hello">
    <h1>Cached components</h1>
    <p>Look at the source code and see how the timestamp is not reloaded before 10s after refreshing the page.</p>
    <p>Timestamp: {{ date }}</p>
  </div>
</template>

<script>
  name: 'HelloWorld',
  serverCacheKey() {
    // Will change every 10 secondes
    return Math.floor(Date.now() / 10000)
  },
  data() {
    return { date: Date.now() }
  }
}
</script>
```
3. Run ssr:serve command

**Additional context**

Please indicate versions of:
```
node: v11.1.0
vue-cli: 3.6.3
uvue: v0.1.0-alpha.18
os: ubuntu 18.04
browser: chrome 73
```

**Comment by chymz** [comment]
Good catch, I'll fix this soon!

**Comment by academici** [comment]
Is it possible to define LRU cache options in server config
What do you think?

**Comment by chymz** [comment]
Actually you can already override this setting in your `server.config.js` file (and btw fix your current issue)

```js
export default {
  renderer: {
    cache: lruCache(/* your options */),
  }
};
```

**Comment by chymz** [comment]
Fixed in `0.1.0-alpha.19`
